### PR TITLE
Add project: modal with instructions about adding a config file

### PR DESCRIPTION
**Pull request by humitos**
This adds a button "I need help" that shows a modal with more instructions about how to add the required `.readthedocs.yaml`.

I will need some help with the copy/instructions we want to show the customer.

Screenshot_2024-11-05_15-52-03

Closes https://github.com/readthedocs/ext-theme/issues/445

### CODE REVIEW

File: readthedocsext/theme/templates/projects/import_base.html [hunk 1]
```diff
@@ -81,12 +81,45 @@ <h2 class="ui small header">{% trans "Help topics" %}</h2>
...
+                          <li>Merge the pull request in your project.</li>
+                        </ol>
+                        <p>
+                          Once that's done Read the Docs will be able to build the documentation for your project.
+                        </p>
+                      </div>
+                      <div class="actions">
+                        <form method="post" action="{{ form_url }}">
+                          <div class="ui cancel button">{% trans "Close" %}</div>
+                        </form>
```
Comment (stsewd) [inline readthedocsext/theme/templates/projects/import_base.html:113]: This doesn't need to be a form

File: readthedocsext/theme/templates/projects/import_base.html [hunk 2]
```diff
@@ -81,12 +81,43 @@ <h2 class="ui small header">{% trans "Help topics" %}</h2>
...
                   {% endif %}
 
                   {% if wizard.steps.next %}
                     <input class="ui button primary" type="submit" value="{% trans "Next" %}" />
-                  {% else %}
+                  {% endif %}
+
+                  {% if wizard.steps.current == "config" %}
+                    <div class="ui small modal" data-modal-id="i-need-help">
+                      <div class="header">Add configuration file to your project</div>
```
Comment (agjohnson) [inline readthedocsext/theme/templates/projects/import_base.html:95]: All the strings here are missing localization

```suggestion
                      <div class="header">{% trans "Add a configuration file to your project" %}</div>
```

File: readthedocsext/theme/templates/projects/import_base.html [hunk 3]
```diff
@@ -81,12 +81,43 @@ <h2 class="ui small header">{% trans "Help topics" %}</h2>
...
+                      <div class="content">
+                        <p>These are the steps you need to follow:</p>
+                        <ol>
+                          <li>
+                            Create a <code>.readthedocs.yaml</code> in your project.
```
Comment (agjohnson) [inline readthedocsext/theme/templates/projects/import_base.html:100]:
```suggestion
                            Create a <code>.readthedocs.yaml</code> file in your repository.
```

File: readthedocsext/theme/templates/projects/import_base.html [hunk 4]
```diff
@@ -81,12 +81,43 @@ <h2 class="ui small header">{% trans "Help topics" %}</h2>
...
+                          </li>
+                          <li>Copy and paste the content of the example.</li>
```
Comment (agjohnson) [inline readthedocsext/theme/templates/projects/import_base.html:102]:
```suggestion
                          <li>Copy the contents of one of our examples to your project's <code>.readthedocs.yaml</code>.</li>
```

File: readthedocsext/theme/templates/projects/import_base.html [hunk 5]
```diff
@@ -81,12 +81,43 @@ <h2 class="ui small header">{% trans "Help topics" %}</h2>
...
+                          <li>Open a pull request with these changes.</li>
+                          <li>Merge the pull request in your project.</li>
```
Comment (agjohnson) [inline readthedocsext/theme/templates/projects/import_base.html:104]: We could also avoid being specific here about pull requests. There is nothing on our side that needs this to be a pull request and users might not understand this.

```suggestion
                          <li>Merge this change to your project's repository.</li>
```

File: readthedocsext/theme/templates/projects/import_base.html [hunk 6]
```diff
@@ -81,12 +81,43 @@ <h2 class="ui small header">{% trans "Help topics" %}</h2>
...
+                        </ol>
+                        <p>
+                          Once that's done Read the Docs will be able to build the documentation for your project.
```
Comment (agjohnson) [inline readthedocsext/theme/templates/projects/import_base.html:107]:
```suggestion
                          Once that's done Read the Docs will be able to build your project's documentation.
```

**Review by agjohnson — APPROVED** [review]
I think we'll eventually want to show some commands here, but this is a great start. Just some notes on copy for now.

**Pull request merged**